### ISSUE TITLE
PyInstaller created kivy app stops on RecycleView

### ISSUE THREAD

**Issue by rjungbeck** [pyinstaller/pyinstaller#2684]
Python: Python 3.6.1 32 bit
OS: Windows 10 64
Kivy: 1.10.0
Kivy installation method: pip (including kivy.deps,angle, kivy.deps.glew, kivy.deps.gstreamer)
PyInstaller: 3.3.dev0-gabfc806 (cause release version does not support Python 3.6)
Description

Application works when started without PyInstaller
Application starts when packaged, but aborts when RecyleView is on opened from App

Similar (unconnected to my app) is on Stackoverflow https://stackoverflow.com/questions/45068125/recycleview-causing-kivy-executable-to-crash

I have opened the same issue on Kivy: https://github.com/kivy/kivy/issues/5276

#### Spec file

````python

# -*- mode: python -*-

from kivy.deps import sdl2, glew

block_cipher = None

options = [ ('v', None, 'OPTION') ]


a = Analysis(['meltflow.py'],
             pathex=['C:\\Users\\ruediger.rj-w8\\PycharmProjects\\meltflow'],
             binaries=[],
             datas=[("*.json", "."),
                ("*.kv", "."),
                ('C:\\Users\\ruediger.rj-w8\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\kivy\\data\\*', "data"),
                ('C:\\Users\\ruediger.rj-w8\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\kivy\\data\\fonts\\*', "data/fonts"),
                ('C:\\Users\\ruediger.rj-w8\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\kivy\\data\\glsl\\*', "data/glsl"),
                ('C:\\Users\\ruediger.rj-w8\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\kivy\\data\\images\\*', "data/images"),
                ('C:\\Users\\ruediger.rj-w8\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\kivy\\data\\keyboards\\*', "data/keyboards"),
                ('C:\\Users\\ruediger.rj-w8\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\kivy\\data\\logo\\*', "data/logo"),
                ("locale/de/LC_MESSAGES/*", "locale/de/LC_MESSAGES"),
                ("locale/en/LC_MESSAGES/*", "locale/en/LC_MESSAGES"),
                ("rst/*", "rst"),
                ("artwork", "artwork"),
                ("*.png", "."),
                ("data/log/dummy.txt", "data/log")],
             hiddenimports=["kivy.uix.recycleview.__init__."],
             hookspath=[],
             runtime_hooks=[],
             excludes=[],
             win_no_prefer_redirects=False,
             win_private_assemblies=False,
             cipher=block_cipher)
pyz = PYZ(a.pure, a.zipped_data,
             cipher=block_cipher)
exe = EXE(pyz,
          a.scripts,
          options,
          exclude_binaries=True,
          name='meltflow',
          debug=False,
          strip=False,
          upx=True,
          console=True )
coll = COLLECT(exe,
               a.binaries,
               a.zipfiles,
               a.datas,
                *[Tree(p) for p in (sdl2.dep_bins + glew.dep_bins)],
               strip=False,
               upx=True,
               name='meltflow')
````

The hidden_input was included to see, if it would solve the problem, because the abort message pointed to that "module" (with trailing dot). Looks like it didn't change a thing.

#### Log

````

Traceback (most recent call last):
  File "C:\Users\ruediger.rj-w8\PycharmProjects\meltflow\meltflow.py", line 277, in main
    app.run()
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\app.py", line 828, in run
    runTouchApp()
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\base.py", line 504, in runTouchApp
    EventLoop.window.mainloop()
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\core\window\window_sdl2.py", line 663, in mainloop
    self._mainloop()
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\core\window\window_sdl2.py", line 405, in _mainloop
    EventLoop.idle()
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\base.py", line 342, in idle
    self.dispatch_input()
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\base.py", line 327, in dispatch_input
    post_dispatch_input(*pop(0))
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\base.py", line 293, in post_dispatch_input
    wid.dispatch('on_touch_up', me)
  File "kivy\_event.pyx", line 718, in kivy._event.EventDispatcher.dispatch (kivy\_event.c:8191)
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\uix\behaviors\button.py", line 179, in on_touch_up
    self.dispatch('on_release')
  File "kivy\_event.pyx", line 718, in kivy._event.EventDispatcher.dispatch (kivy\_event.c:8191)
  File "C:\Users\ruediger.rj-w8\PycharmProjects\meltflow\menu.py", line 149, in on_release
    self.doRelease()
  File "C:\Users\ruediger.rj-w8\PycharmProjects\meltflow\menu.py", line 155, in doRelease
    self.sequencer.runSequence(self.cmd["action"], done=self.doneHandler, error=self.error)
  File "C:\Users\ruediger.rj-w8\PycharmProjects\meltflow\sequencer.py", line 165, in runSequence
    self.curSequence=Sequence(self, seq, done=done, error=error)
  File "C:\Users\ruediger.rj-w8\PycharmProjects\meltflow\sequencer.py", line 335, in __init__
    self.step()
  File "C:\Users\ruediger.rj-w8\PycharmProjects\meltflow\sequencer.py", line 341, in step
    self.execute()
  File "C:\Users\ruediger.rj-w8\PycharmProjects\meltflow\sequencer.py", line 377, in execute
    cont=obj.run()
  File "C:\Users\ruediger.rj-w8\PycharmProjects\meltflow\seqentity.py", line 117, in run
    entityselect.EntitySelect(self.sequencer, self.curCmd, done=self.completed, error=self.error)
  File "C:\Users\ruediger.rj-w8\PycharmProjects\meltflow\entityselect.py", line 39, in __init__
    XPopup.__init__(self, title=gettext(cmd["title"]), **kwargs)
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\uix\modalview.py", line 152, in __init__
    super(ModalView, self).__init__(**kwargs)
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\uix\anchorlayout.py", line 68, in __init__
    super(AnchorLayout, self).__init__(**kwargs)
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\uix\layout.py", line 76, in __init__
    super(Layout, self).__init__(**kwargs)
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\uix\widget.py", line 345, in __init__
    Builder.apply(self, ignored_consts=self._kwargs_applied_init)
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\lang\builder.py", line 451, in apply
    self._apply_rule(widget, rule, rule, ignored_consts=ignored_consts)
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\lang\builder.py", line 567, in _apply_rule
    self._apply_rule(child, crule, rootrule)
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\lang\builder.py", line 526, in _apply_rule
    cls = Factory_get(cname)
  File "C:\users\ruediger.rj-w8\AppData\Local\Programs\Python\Python36-32\lib\site-packages\kivy\factory.py", line 139, in __getattr__
    module = __import__(name=item['module'], fromlist='.')
  File "<frozen importlib._bootstrap>", line 1009, in _handle_fromlist
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 936, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 948, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'kivy.uix.recycleview.__init__.'

```

**Comment by rjungbeck** [comment]
Issue is solved via kivy/kivy#5276

**Comment by htgoebel** [comment]
So this has been a kivy problem.
